src/blant-fundamentals.h

```diff
@@ -18,10 +18,14 @@
 
 // MAX_K is the maximum number of nodes in a graphlet that is supported by BLANT when using a fixed lookup table (as
 // opposed to one that uses associaive arrays).  Maximum value is 7 with self-loops, 8 without.
-#define MAX_K (8-SELF_LOOPS) // NOTE that this is for BLANT; the canon_map creation codes can use different MAXK
+#define MAX_K (9-SELF_LOOPS) // NOTE that this is for BLANT; the canon_map creation codes can use different MAXK
 
 // maximum number of entries in the canon_map (lookup table), which is 2^(k choose 2) without self-loops
+#if MAX_K <= 8
 #define maxBk (1 << (MAX_K*(MAX_K-1)/2 + MAX_K*SELF_LOOPS))
+#else
+#define maxBk (1 << (8*(8-1)/2 + 8*SELF_LOOPS))
```

Comment:
Is there a reason 8 is hard-coded? It shouldn't be. MAX_K should work whether it's <=8 or >8.

Reply:
Because for k>8 we are using a different method so we don't want to create a big array (_K) for k>8. 
In fact we can't, that's why we are using a different method